nautobot/utilities/filters.py

```diff
@@ -1,35 +1,24 @@
-from collections import OrderedDict
-from copy import deepcopy
 import logging
 import uuid
+from collections import OrderedDict
+from copy import deepcopy
 
-from django import forms
+import django_filters
+from django import forms as djforms
 from django.conf import settings
 from django.core.validators import MaxValueValidator
 from django.db import models
 from django.forms.utils import ErrorDict, ErrorList
-
-import django_filters
 from django_filters.constants import EMPTY_VALUES
 from django_filters.utils import get_model_field, resolve_field
-
 from mptt.models import MPTTModel
-from tree_queries.models import TreeNode
-
-from nautobot.dcim.fields import MACAddressCharField
-from nautobot.dcim.forms import MACAddressField
-from nautobot.extras.models import Tag
-from nautobot.utilities.constants import (
-    FILTER_CHAR_BASED_LOOKUP_MAP,
-    FILTER_NEGATION_LOOKUP_MAP,
-    FILTER_NUMERIC_BASED_LOOKUP_MAP,
-)
-from nautobot.utilities.forms.fields import MultiMatchModelMultipleChoiceField, MultiValueCharField
-from nautobot.utilities.utils import flatten_iterable
-
-
 from taggit.managers import TaggableManager
+from tree_queries.models import TreeNode
 
+from nautobot.dcim import fields as dcim_fields
+from nautobot.dcim import forms as dcim_forms
```

Comment:
Should these two be merged?

```suggestion
from nautobot.dcim import fields as dcim_fields, forms as dcim_forms
```

Reply:
I think that when aliasing module names it's easier to read if they're on separate lines. Black, isort and flake8 don't complain about it (with the isort config above)

Reply:
Fair enough. :-)